api/slotchange.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Flagging @Elchi3 here — is this the correct way to add an actual event to BCD (as opposed to a DOM event object?)

Reply:
I mean, shouldn't we put events in a separate events folder? And maybe have the JSON structure as events.slotchange or similar?

Reply:
Yes, please hold on with events for now. See the discussion in https://github.com/mdn/browser-compat-data/issues/935.